dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -184,6 +184,9 @@ protected static String extractDoiFromDoiUrl(String doiUrl) {
         try {
             URI uri = new URI(doiUrl);
             doi = StringUtils.stripStart(uri.getPath(), "/");
+            if (doi.startsWith("doi/")) {
```

Comment:
I think this would fix future DOIs and then we would have to do a database migration to change the past DOIs. I didn't realize the migrations happen in this repo so I'll add that onto this PR

Reply:
The `conceptdoi` appears to be included in raw form in the zenodo response, could we retrieve it from there and avoid the url manipulation?  https://zenodo.org/api/records/11093965

Reply:
If we do stick with extracting the concept DOI from the link url, and we need a cue as to where the Zenodo DOI begins  in a given string, Zenodo DOIs always start with `10.5281/zenodo.` in the current Zenodo DOI generation scheme.

Reply:
I changed it so we use the `conceptdoi` from the zenodo response.